### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gianm**
- Scala 2.10.5
- Druid 0.8.1
- Finagle 6.25.0
- Jackson 2.6.0
- scala-util 1.11.3
- slf4j 1.7.12

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -51,14 +51,14 @@ releaseSettings
 ReleaseKeys.publishArtifactsAction := PgpKeys.publishSigned.value
 
 val jacksonOneVersion = "1.9.13"
-val jacksonTwoVersion = "2.4.4"
-val druidVersion = "0.7.0"
-val finagleVersion = "6.24.0"
-val twitterUtilVersion = "6.23.0"
+val jacksonTwoVersion = "2.6.0"
```
Comment (xvrl) [inline build.sbt]: should we use latest 2.6.3 if we're already upgrading it?

Comment (gianm) [inline build.sbt]: sure

Comment (gianm) [inline build.sbt]: 2.6.1 is the latest jackson-module-scala, will use that one instead

**Comment by gianm** [comment]
@xvrl updated jackson

**Comment by xvrl** [comment]
:+1:

**Pull request merged**